SteamKit2/SteamKit2/Steam/SteamClient/CallbackMgr/CallbackMsg.cs

```diff
@@ -6,22 +6,10 @@
 
 namespace SteamKit2
 {
-    /// <summary>
-    /// A callback message
-    /// </summary>
-    public interface ICallbackMsg
```

Comment:
I don't see why this interface was removed, unrelated to the changes in this PR?

Reply:
This is opinionated, but either the interface or the abstract class should be removed.
CallbackMsg is the sole inheritor of ICallbackMsg and just adds further complexity as a user won't know which one to use as they have the same purpose.

Reply:
I understand that, but that seems outside of the scope of this PR. Want to make a separate one just for this change?